cli/tests/056_make_temp_file_write_perm.ts

```diff
@@ -1,5 +1,6 @@
-const path = await Deno.makeTempFile({ dir: "./subdir/" });
-if (path.startsWith(Deno.cwd())) {
+const sep = Deno.build.os == "windows" ? "\\" : "/";
+const path = await Deno.makeTempFile({ dir: `.${sep}subdir` });
+if (path.startsWith(`.${sep}subdir${sep}`)) {
```

Comment:
The branch for windows is rather unfortunate. Is it just to make the `startsWith` work?

Reply:
Cleaned up.